tasks.rb

```diff
@@ -3823,6 +3824,14 @@ def apply_hpxml_modification(hpxml_file, hpxml)
     hpxml.hvac_controls[0].cooling_setup_temp = 80
     hpxml.hvac_controls[0].cooling_setup_hours_per_week = 6 * 7
     hpxml.hvac_controls[0].cooling_setup_start_hour = 9 # 9am
+  elsif ['base-hvac-onoff-thermostat-deadband.xml'].include? hpxml_file
+    hpxml.header.timestep = 1
+    hpxml.header.temperature_capacitance_multiplier = 4.0
+    hpxml.hvac_controls[0].onoff_thermostat_deadband = 2.0
+    hpxml.heat_pumps[0].heating_capacity = 10000
+    hpxml.heat_pumps[0].heating_capacity_17F = 5900.0
+  elsif ['base-atticroof-cathedral.xml'].include? hpxml_file
+    hpxml.hvac_distributions[0].conditioned_floor_area_served = 2700
```

Comment:
What's this change for?

Reply:
I don't think I made this change, must be something from resolving conflicts. Will remove on next commit.